feature-libs/organization/account-summary/components/account-summary/details/header/header.component.ts

```diff
@@ -0,0 +1,119 @@
+import { Component, OnInit } from '@angular/core';
+import { Address, TranslationService } from '@spartacus/core';
+import {
+  AccountSummaryDetails,
+  AccountSummaryFacade,
+} from '@spartacus/organization/account-summary/root';
+import { Card } from '@spartacus/storefront';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+const notApplicable = 'n/a';
+
+@Component({
+  selector: 'cx-account-summary-header',
+  templateUrl: './header.component.html',
+})
+export class HeaderComponent implements OnInit {
+  headerDetails$: Observable<AccountSummaryDetails>;
+
```

Comment:
`headerDetails$ = this.accountSummaryFacade.getAccountSummary();`, so, `ngOnInit` can be removed.